reflex/components/radix/themes/base.py

```diff
@@ -11,7 +11,9 @@
 
 LiteralAlign = Literal["start", "center", "end", "baseline", "stretch"]
 LiteralJustify = Literal["start", "center", "end", "between"]
-LiteralSize = Literal["1", "2", "3", "4", "5", "6", "7", "8", "9"]
+LiteralSize = Literal[
+    "1", "2", "3", "4", "5", "6", "7", "8", "9", 1, 2, 3, 4, 5, 6, 7, 8, 9
```

Comment:
i don't really like this, because now the autocomplete list seems to have duplicates in it; the generated docs will show both the str and the int values, and it's kind of clunky.

i would suggest just keeping the str values, but doing the conversion to `str` under the hood if the user provides an `int`

Reply:
I had it this way because I thought the idea was to support strs and ints in the API?